scripts/install_environment.sh

```diff
@@ -21,9 +21,37 @@ function install_libvirt() {
         libvirt-devel \
         libvirt-daemon-kvm \
         qemu-kvm \
-        libgcrypt \
-        swtpm \
-        swtpm-tools
+        libgcrypt
+
+    # TODO: get swtpm RPMs from EPEL as soon as 0.7.1 or later lands there.
+    # based on: https://github.com/stefanberger/swtpm/wiki#compile-and-install-on-linux
+   dnf install -y \
```

Comment:
interesting
if it doesn't add too much time I wouldn't mind going that direction

Reply:
worked for me locally but seems to fail here. I suspect it might be missing EPEL. looking into it.

Reply:
Indeed, EPEL was the issue here.